WordPressAuthenticator/NUX/NUXButton.swift

```diff
@@ -69,7 +82,10 @@ import WordPressUI
     open override func awakeFromNib() {
         super.awakeFromNib()
         configureAppearance()
-        activityIndicator.style = .gray
+        guard #available(iOS 13, *) else {
+            activityIndicator.style = .gray
```

Comment:
Do we need this? Should we just initialise the indicator with a `.gray` appearance instead of white?